app/views/records/show_fields/_rights.html.erb

```diff
@@ -1,4 +1,4 @@
 <% record.rights.each do |r| %>
-  <%=link_to_field('rights',r, Sufia.config.cc_licenses_reverse[r])%> <%= iconify_auto_link(r,false) %><br />
+  <%=link_to_field('rights',r, Sufia.config.cc_licenses_reverse[r])%><% if is_url?(r) %> <%=iconify_auto_link(r,false) %><% end %><br />
```

Comment:
Maybe a compromise approach?  Make a new helper that is for linking licenses?  Mainly I don't want to see a lot of conditionals in the views.